CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Added: `at-rule-name-newline-after` rule.
 - Fixed: `function-url-quotes` ignores values containing `$sass` and `@less` variables.
 - Fixed: selector-targeting rules ignore Less mixins and extends.
+- Fixed: no-unsupported-browser-features message now includes feature name
```

Comment:
Needs backticks around rule name and a period at the end of the sentence. Also, I'd consider this `Added` rather than `Fixed`. Right?